### DISCUSSION TITLE
Cost change DAE

### DISCUSSION THREAD

**Discussion by crimson10910** [misthero/dnd5e-spellpoints#113] [Q&A]
Hello! I have a question. 
I'd like to change the costs for an individual player using DAE. Are there any flags or something else to implement this?

**Comment by misthero** [comment]
you can probably use a macro for that and activate it with dae.

I don't know what you want to do exactly, but for example this is a macro to decrease the item uses by 2:

![image](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/7f720394-29db-414a-a0c4-ea8853ff3d2f)
